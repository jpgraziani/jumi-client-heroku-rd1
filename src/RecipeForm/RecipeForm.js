import React from 'react'

export default function RecipeForm(props) {
  const { className, ...otherProps } = props
  return (
    <form
      className={['Recipe-form', className].join(' ')}
      action='#'
      {...otherProps}
    />
  )
}
