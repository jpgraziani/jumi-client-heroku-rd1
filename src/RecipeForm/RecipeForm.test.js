import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import RecipeForm from './RecipeForm'


describe(`RecipeForm component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  }

  it('renders a form.RecipeForm by default', () => {
    const wrapper = shallow(<RecipeForm />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the RecipeForm given props', () => {
    const wrapper = shallow(<RecipeForm {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})