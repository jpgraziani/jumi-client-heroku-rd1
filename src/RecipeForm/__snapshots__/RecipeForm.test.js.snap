// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeForm component renders a form.RecipeForm by default 1`] = `
<form
  action="#"
  className="Recipe-form "
/>
`;

exports[`RecipeForm component renders the RecipeForm given props 1`] = `
<form
  action="#"
  className="Recipe-form test-class-name"
  data-other="test-other-prop"
>
  <p>
    test children
  </p>
</form>
`;
